ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -125,6 +142,70 @@ export default {
         },
       },
     },
+    {
+      id: 'DelubrumSav Seeker Dead Iron',
+      // Headmarkers are randomized, so use the tether instead.
+      netRegex: NetRegexes.tether({ target: 'Trinity Seeker', id: '01DB' }),
+      condition: (data, matches) => matches.source === data.me,
+      alarmText: (data, _, output) => output.earthshaker(),
+      outputStrings: {
+        earthshaker: Outputs.earthshakerOnYou,
+      },
+    },
+    {
+      id: 'DelubrumSav Seeker Iron Splitter',
+      netRegex: NetRegexes.startsUsing({ source: ['Trinity Seeker', 'Seeker Avatar'], id: '5AC0' }),
+      preRun: (data) => delete data.ironSplitter,
+      promise: async (data, matches) => {
+        const seekerData = await window.callOverlayHandler({
+          call: 'getCombatants',
+          ids: [parseInt(matches.sourceId, 16)],
+        });
+
+        if (seekerData === null) {
+          console.error(`Iron Splitter: null data`);
+          return;
+        }
+        if (!seekerData.combatants) {
+          console.error(`Iron Splitter: null combatants`);
+          return;
+        }
+        if (seekerData.combatants.length !== 1) {
+          console.error(`Iron Splitter: expected 1, got ${seekerData.combatants.length}`);
+          return;
+        }
+
+        const seeker = seekerData.combatants[0];
+        const x = seeker.PosX - seekerCenterX;
+        const y = seeker.PosY - seekerCenterY;
+        data.splitterDist = Math.hypot(x, y);
+      },
+      alertText: (data, _, output) => {
+        if (data.splitterDist === undefined)
+          return;
+
+        // All 100 examples I've looked at only hit distance=10, or distance=~14
+        // Guessing at the other distances, if they exist.
+        //
+        // blue inner = 0?
+        // white inner = 6?
+        // blue middle = 10
+        // white middle = 14
+        // blue outer = 18?
+        // white outer = 22?
+
+        const isWhite = Math.floor(data.splitterDist / 4) % 2;
+        return isWhite ? output.goBlue() : output.goWhite();
+      },
+      outputStrings: {
+        goBlue: {
+          en: 'Go Blue',
```

Comment:
@Legends0 Do you have any opinions about these triggers? Do other folks who aren't me say blue/white or does it need to say like "Go Blue Stone" or "Go White Sand" or some other variation?

Reply:
My group has stated "Stand on Sand" or "Stand on Brick".

Reply:
I think 'Be on' will be a better phrase to use as it gets you to look down at your feet rather than looking around for someone to go to. The noun used may vary, sand seems pretty universal to use, but brick caught me off the first time it was used but looking at the ground more closely it made sense after hearing it.

Reply:
My worry here is that the thing you have to do is several words in, and the timing is pretty tight.

I'd almost say "Blue stone" or "white sand" on its own, unless there's some way to say like "blue stone (stand on it!)" that sounds less awkward.  I think people in savage should know what this mechanic is, and I think there's some advantage to getting out the important word as quickly as possible.